### DISCUSSION TITLE
T02 contract terms, OPP-030 and OPP-031 and associated codelist

### DISCUSSION THREAD

**Discussion by odscjen** [OP-TED/eForms-SDK#230] [Q&A] [answered]
The 1.3.2 SDK guidance gives the xpath for these two terms as:

OPP-030: efac:ContractTerm​/efbc:TermCode[@listName='contract-term']
OPP-031: efac:ContractTerm[not(efbc:TermCode​/text()='all-rev-tic')][efbc:TermCode​/@listName='contract-term']​/efbc:TermDescription

But I can't find a codelist titled "contract-term". Should it be ["contract-detail"](https://github.com/OP-TED/eForms-SDK/blob/develop/codelists/contract-detail.gc), this is what the example XML [t02-ESP.xml](https://github.com/OP-TED/eForms-SDK/blob/develop/examples/notices/t02_ESP.xml) uses?

**Comment by YvesJo** [comment, marked as answer]
Indeed, the guidance was written a long time ago while another codelist name ("contract-term") was foreseen. In the meantime, the name of the list changed ("contract-detail") and the guidance has unfortunately not been updated, this does not however remove the illustration value of the example in the guidance, even if it is not exactly what it should be.
